### DISCUSSION TITLE
Information about the docker volume created by the data-importer

### DISCUSSION THREAD

**Discussion by SergioRAgostinho** [firefly-iii/firefly-iii#6675] [Firefly III] [answered]
I noticed the data importer docker container is making use of a volume. However, the [instructions](https://docs.firefly-iii.org/data-importer/install/docker/) regarding its installation make no mention to it. I am struggling the find the Dockerfile used to create this container inside the repo, so currently my information about what this volume is and what it is used for, is a bit limited. 

Inspecting the container shows me 
```json
"Mounts": [
            {
                "Type": "volume",
                "Name": "a7f286fec205da156558e0d1ca4ba10dd7d7a05739bee7903768a1593e297795",
                "Source": "/var/lib/docker/volumes/a7f286fec205da156558e0d1ca4ba10dd7d7a05739bee7903768a1593e297795/_data",
                "Destination": "/var/www/html/storage/upload",
                "Driver": "local",
                "Mode": "z",
                "RW": true,
                "Propagation": ""
            }
        ],
```

which means it's being mounted to `/var/www/html/storage/upload`. What is being stored there? Since this is a volume I assumed we are interested in preserving data between reboots, instead of creating an unnamed volume every time. Should instruction for that be added to the documentation?

Best regards

**Comment by JC5** [comment, marked as answer]
The volume is defined in the base image that's shared by both Firefly III and the Data Importer. There's currently nothing being stored, but this may change in the future.

**Comment by SergioRAgostinho** [comment]
> The volume is defined in the base image that's shared by both Firefly III and the Data Importer. 

Where can I find more information about this image?
Edit: Here https://dev.azure.com/Firefly-III/_git/MainImage 

(In the future, ) Is `/var/www/html/storage/upload` meant to be shared by both Firefly III and the Data Importer? Just for me to understand if I need to created a separate named volume for both containers of if they should share the same.

**Reply by JC5** [reply]
They should not share the same volume, not now and not in the future.
